rescue_from.gemspec

```diff
@@ -18,7 +18,16 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  spec.add_dependency "rails", '>= 3.1', '< 5.0'
+  spec.add_dependency "exception_notification", '>= 4.1', '< 5.0'
+
+  spec.add_development_dependency "pg", '~> 0.18.3'
   spec.add_development_dependency "bundler", "~> 1.10"
   spec.add_development_dependency "rake", "~> 10.0"
-  spec.add_development_dependency "rspec"
+  spec.add_development_dependency "rspec-rails", '~> 3.3.3'
+  spec.add_development_dependency "pry-nav", '~> 0.2.4'
+  spec.add_development_dependency "pry-rails", '~> 0.3.4'
+  spec.add_development_dependency "apipie-rails", '~> 0.3.4'
```

Comment:
The exceptions get initialized when they're raised, my main test is an integration controller test, and this one is in the original list from tutor server